### PR TITLE
Book detail 4 - Badge

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary
- Added badge and new styles to book detail
- Added new transform mixin

## Screenshot
Captura de Pantalla 2021-05-11 a la(s) 11 34 16 a  m

## Trello Card
https://trello.com/c/MWfuNTDx/29-detalle-de-libro-iv-badge

### CODE REVIEW

File: books/scss/detail.scss [hunk 1]
```diff
@@ -6,9 +6,26 @@
...
+    .book-box-badge {
+      height: 7.4rem;
+      position: absolute;
+      right: -1.4rem;
+      top: -1rem;
+      transform: rotate(12deg);
+      -webkit-transform: rotate(12deg);
+      -moz-transform: rotate(12deg);
+      -ms-transform: rotate(12deg);
+      -o-transform: rotate(12deg);
```
Comment (mcallegari10) [inline books/scss/detail.scss]: There's no need to fill all prefixes. In the projects, the autoprefixers will do the job 😄

File: books/scss/detail.scss [hunk 2]
```diff
@@ -6,9 +6,26 @@
...
+    .book-box-cover {
+      height: 368px;
+      width: 261px;
+    } 
+
+    .book-box-badge {
+      height: 7.4rem;
+      position: absolute;
+      right: -1.4rem;
+      top: -1rem;
```
Comment (mcallegari10) [inline books/scss/detail.scss]: Always try to avoid negative values to absolute positioning. Change it so you have positive please

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: books/scss/detail.scss (hunk 2 above)
Comment (Julianhm9612) [reply books/scss/detail.scss]: Have been improved

File: books/scss/detail.scss (hunk 1 above)
Comment (Julianhm9612) [reply books/scss/detail.scss]: Unnecessary lines were removed

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**